core/variable.cpp

```diff
@@ -336,9 +327,6 @@ Variable mean(const VariableConstProxy &var, const Dim dim) {
     summed = summed * makeVariable<double>(scale);
```

Comment:
Since you no longer need the scale below, you could even remove the duplicate `makeVariable<double>(scale)`, e.g.
```C++
auto scale = makeVariable<double>(1.0 / static_cast<double>(var.dims()[dim]));
if (isInt(var.dtype()))
    summed = summed * scale;
else
    summed *= scale;
```